service-front/content/guidance/People_Correspondent.md

```diff
@@ -1,5 +1,9 @@
 # Correspondent
 
+<div class="moj-banner moj-banner__message">
+    <p class="bold-small"><a href="https://www.gov.uk/guidance/making-and-registering-an-lpa-during-the-coronavirus-outbreak#think-carefully-about-who-should-be-sent-the-registered-lpa">Read our guidance</a>  getting on being a correspondent during the corona virus outbreak </p>
```

Comment:
"Read our guidance getting on being..." doesn't make sense and the story says "Read our guidance on getting on being..." which isn't any better. I assume there was a copy paste error when writing the story and it was supposed to be "Read our guidance on being...".

Reply:
Corrected as "Read our guidance on being...".